openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2215,7 +2215,7 @@ The following is a non-normative example of an object containing the `credential
 
 The following additional claims are defined for authorization details of type `openid_credential` and this Credential Format.
 
-* `doctype`: OPTIONAL. String as defined in (#server-metadata-mso-mdoc). This claim contains the type value the Wallet requests authorization for at the Credential Issuer. It MUST only be present if the `format` claim is present. It MUST not be present otherwise. 
+* `doctype`: REQUIRED. String as defined in (#server-metadata-mso-mdoc). This claim contains the type value the Wallet requests authorization for at the Credential Issuer. It MUST only be present if the `format` claim is present. It MUST not be present otherwise. 
```

Comment:
It can't be both "REQUIRED" and "MUST only be present if the `format` claim is present"?

Reply:
Agreed, the other option we could do is have something like CONDITIONAL for these sorts of elements?

Reply:
we do not use conditional in openid4vc specs, because it's not part of rfc 2119

Reply:
```suggestion
* `doctype`: REQUIRED when the `format` claim is present. MUST NOT be present otherwise. String as defined in (#server-metadata-mso-mdoc) that contains the type value the Wallet requests authorization for at the Credential Issuer.
```

Reply:
```suggestion
* `doctype`: REQUIRED if the `format` claim is present with a value of `mso_mdoc`. It MUST NOT be present otherwise. String as defined in (#server-metadata-mso-mdoc) that contains the type value the Wallet requests authorization for at the Credential Issuer.
```

Shouldn't we be more specific, also the example authorisation details object needs the format claim.

Reply:
I've applied my suggestion above consistently, only other change is to adjust the examples which I'm happy to do.

Reply:
thank you!